sycl/doc/extensions/README.md

```diff
@@ -1,3 +1,34 @@
 # Extensions
 
-This is where documents can be found that propose extensions to the SYCL specification.
+The directory contains documents that describe DPC++ extensions to SYCL
+specification.
+
+DPC++ extensions status:
+
+|  Extension  |    Status   |   Comment   |
+|-------------|:------------|:------------|
+| [SYCL_INTEL_bitcast](Bitcast/SYCL_INTEL_bitcast.asciidoc) | Proposal | |
+| [C and C++ Standard libraries support](C-CXX-StandardLibrary/C-CXX-StandardLibrary.rst) | Partially supported | |
+| [SYCL_INTEL_data_flow_pipes](DataFlowPipes/data_flow_pipes.asciidoc) | Supported | |
```

Comment:
One clarification here:
kernel_host_pipe_support part is not implemented (and probably won't be implemented), since it's believed, that USM can help in data streaming from host to FPGA device.
So I suggest leaving it as "Partially supported".